### PR TITLE
Nicolas martin castellanos

### PR DESCRIPTION
**Pull request by NicolasMartin23**
send uploading magazine to PR

### CODE REVIEW

File: nicolas-martin-castellanos/tarea-1/magazine1/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="box">
+      <div class="jordan">
```
Comment (emolinaglobant) [inline nicolas-martin-castellanos/tarea-1/magazine1/index.html]: @NicolasMartin23 , remember you should be using semantic tags and not divs

Comment (NicolasMartin23) [reply nicolas-martin-castellanos/tarea-1/magazine1/index.html]: I made the correction and forwarded to PR.

File: nicolas-martin-castellanos/tarea-1/magazine1/style.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+.box{
+    width: 100%;
+    height: 100vh;
+    background-color: rgba(0, 0, 0, 0.4);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.jordan{
+    z-index: 1;
```
Comment (emolinaglobant) [inline nicolas-martin-castellanos/tarea-1/magazine1/style.css:21]: @NicolasMartin23 , please make sure all CSS properties are alphabetically sorted

Comment (NicolasMartin23) [reply nicolas-martin-castellanos/tarea-1/magazine1/style.css:21]: Send Magazine with CSS properties sorted alphabetically to PR.

**Comment by emolinaglobant** [comment]
@NicolasMartin23 , i'm gonna merge the PR  BUT some properties are still without being sorted, and your sorting in descendant order should be asscendant

**Review by emolinaglobant — APPROVED** [review]

**Comment by emolinaglobant** [comment]
@NicolasMartin23 , looks like the semantic tags broke your style
Screen Shot 2022-01-26 at 7 18 24 AM
s

**Pull request merged**